challenge_6/scala/ndsmith3/Ranges.scala

```diff
@@ -0,0 +1,34 @@
+import scala.annotation.tailrec
+
+object Ranges {
+  @tailrec
+  def makeRangeString(answer: List[String])(curr: Array[Int]): List[String] = {
+    val answerPlusNext       = (curr.head.toString :: answer)
+    lazy val nextIsIncrement = curr.head + 1 != curr(1)
```

Comment:
The lazy variables are there to prevent an exception being thrown in the case of curr or answer being empty.